src/view.rs

```diff
@@ -53,7 +53,7 @@ impl ViewKey {
 
     /// Derive the secret to deterministically construct a [`PublicSpendKey`]
     pub fn public_key(&self) -> PublicSpendKey {
-        let A = GENERATOR_EXTENDED * &self.a;
+        let A = GENERATOR_EXTENDED * self.a;
```

Comment:
Same as above

Reply:
It's enabled for the whole repo in `lib.rs`